containers/dart-flutter-android/.devcontainer/Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM debian:stretch-slim
+
+# Hack to get openjdk to install in a container
+RUN mkdir -p /usr/share/man/man1 \
+    && mkdir -p /usr/share/man/man7
+
+# Apt
+RUN apt-get update && apt-get install -y curl wget git xz-utils lib32stdc++6 unzip openjdk-8-jdk-headless
+
+# Android SDK
+RUN wget https://dl.google.com/android/repository/sdk-tools-linux-4333796.zip \
+ &&  mkdir android-sdk && unzip /sdk-tools-linux-4333796.zip -d android-sdk \
+ && rm /sdk-tools-linux-4333796.zip
+ENV ANDROID_HOME="/android-sdk"
+ENV PATH="/android-sdk/tools/bin:/android-sdk/build-tools:/android-sdk/platform-tools:${PATH}"
+
+# SDK manager
+RUN yes | sdkmanager --licenses
+RUN sdkmanager "platforms;android-28" "platform-tools" "build-tools;28.0.3"
+
+# Flutter
+RUN wget https://storage.googleapis.com/flutter_infra/releases/stable/linux/flutter_linux_v1.5.4-hotfix.2-stable.tar.xz \
+ && tar xf flutter_linux_v1.5.4-hotfix.2-stable.tar.xz \
+ && rm flutter_linux_v1.5.4-hotfix.2-stable.tar.xz
```

Comment:
This seems like a fairly old version of Flutter? Current version is https://storage.googleapis.com/flutter_infra/releases/stable/linux/flutter_linux_v1.12.13+hotfix.5-stable.tar.xz